src/nwb_benchmarks/core/_dandi.py

```diff
@@ -16,7 +16,8 @@ def get_s3_url(dandiset_id: str, dandi_path: str) -> str:
     """
     assert len(dandiset_id) == 6, f"The specified 'dandiset_id' ({dandiset_id}) should be the six-digit identifier."
 
-    client = DandiAPIClient()
+    api_url = "https://api-staging.dandiarchive.org/api" if is_staging else "https://api.dandiarchive.org/api"
```

Comment:
```suggestion
    api_url = "https://api-staging.dandiarchive.org/api" if int(dandiset_id) >= 200_000 else "https://api.dandiarchive.org/api"
```

No need for flag, you can tell by the ID

Reply:
Moved this to #61